Source/WebKit/WPEPlatform/CMakeLists.txt

```diff
@@ -74,12 +74,14 @@ set(WPEPlatform_PRIVATE_INCLUDE_DIRECTORIES
 
 set(WPEPlatform_SYSTEM_INCLUDE_DIRECTORIES
     ${GLIB_INCLUDE_DIRS}
+    ${GIO_UNIX_INCLUDE_DIRS}
```

Comment:
This was already added to the DRM makefile, no?

Reply:
Checked, and indeed changing the include is enough. Patch updated.